src/soxr/sox-resampler.cc

```diff
@@ -96,7 +96,14 @@ void SoXResampler::start (int & channels, int & rate)
     if (new_rate == rate)
         return;
 
-    soxr_quality_spec_t q = soxr_quality_spec (aud_get_int ("soxr", "quality"), 0);
+    int quality = aud_get_int ("soxr", "quality");
+    int phase_response = aud_get_int ("soxr", "phase_response_list");
```

Comment:
Please add defaults for the new settings to the SoXResampler::defaults table earlier in the file.  Use "TRUE" or "FALSE" for Booleans, and aud::numeric_string for enumeration values.